lib/test-observable.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const Rx = require('rx');
+const compile = require('test262-compiler');
+const util = require('util');
+
+const scenariosForTest = require('./scenarios.js');
+
+module.exports = function(test262Dir, options = {}) {
+  const prelude = options.prelude || '';
+  const includesDir = options.includesDir;
+  const absolutePaths = (options.paths || [''])
+    .map(relativePath => path.join(test262Dir, relativePath));
+
+  return absolutePaths
+    .map(findFiles)
+    .reduce((accumulator, moreFiles) => accumulator.merge(moreFiles))
+    .map(pathToTestFile)
+    .map(test => compileFile(test, test262Dir, prelude, includesDir))
+    .flatMap(scenariosForTest);
+};
+
+function pathToTestFile(path) {
+  return { file: path, contents: fs.readFileSync(path, 'utf-8')};
```

Comment:
If you rename `path` to `file`:

```js
function pathToTestFile(file) {
  return { file, contents: fs.readFileSync(file, 'utf-8')};
}
```


Additionally, it avoids any future reader confusion with the built-in module named `path` that is also required in this file and used inside a later function.

Reply:
Agreed, but this was solved by an unrelated change